server/database/queries/addData.js

```diff
@@ -0,0 +1,6 @@
+
+const connction = require('../config/db_connection');
+
+const addStudent = (firstname, lastname, Address, IsActive, AmideastId, phonenumber, mobilenumber, level, password) => connction.query('INSERT INTO student(firstname,lastname,address,amideastId,isActive,homePhone,mobilePhone,level,password) values($1,$2,$3,$4,$5,$6,$7,$8,$9) returning *', [firstname, lastname, Address, IsActive, AmideastId, phonenumber, mobilenumber, level, password]);
```

Comment:
@naremanhilles this isn't super important but it can help readability if you use consistent capitalization for all of your variables.  So here, you could use camelCase for everything:
`const addStudent = (firstname, lastname, address, isActive, amideastId, phonenumber, ... `

Reply:
thanks ,i fexed it now